config/routes.rb

```diff
@@ -48,4 +48,10 @@
   end
 
   root 'v0/example#index', module: 'v0'
+
+  require "sidekiq/web"
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    username == ENV["SIDEKIQ_USERNAME"] && password == ENV["SIDEKIQ_PASSWORD"]
+  end if Rails.env.production?
```

Comment:
It will be interesting to see if this might interfere with the basic authentication already built in for dev and staging environments -- having multiple BASIC Authentication in both might cause issues?